program/Cargo.toml

```
@@ -17,12 +17,12 @@ test-sbf = []
 
 [dependencies]
 num_enum = "0.7.2"
-solana-program = "1.18.2"
+solana-program = "2.0.1"
 spl-program-error = "0.4.0"
```

Wanna bump this too?
```suggestion
spl-program-error = "0.5.0"
```